cf/aggregate.py

```diff
@@ -2358,18 +2358,26 @@ def aggregate(
 
             >>> x = cf.aggregate(fl, cells=cf.climatology_cells())
 
+            **Storage of conditions**
+
+            All returned field or domain constructs that have passed
+            dimension coordinate cell conditions will have those
+            conditions stored on the approriate dimension coordinate
```

Comment:
One more typo fix, after I thought to check on any other repeats:

```suggestion
            conditions stored on the appropriate dimension coordinate
```